src/Schema/TypeRegistry.php

```diff
@@ -68,14 +68,17 @@ class TypeRegistry
     /**
      * Map from type names to resolved types.
      *
-     * @var array<string, \GraphQL\Type\Definition\Type>
+     * May contain `null` if the type is known to not exist, this allows
+     * short-circuiting repeated lookups for the same type.
+     *
+     * @var array<string, (\GraphQL\Type\Definition\Type&\GraphQL\Type\Definition\NamedType)|null>
      */
     protected $types = [];
```

Comment:
how can the type be null? 🤔

Reply:
As explained above. I changed the description to hopefully be more clear.